eternalcore-core/src/main/java/com/eternalcode/core/feature/fun/endscreen/EndScreenCommand.java

```diff
@@ -0,0 +1,48 @@
+package com.eternalcode.core.feature.fun.endscreen;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import com.eternalcode.paper.PaperOverlay;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import org.bukkit.entity.Player;
+
+@Command(name = "endscreen", aliases = {"end-screen", "win-screen"})
+@Permission("eternalcore.fun.endscreen")
```

Comment:
```suggestion
@Permission("eternalcore.endscreen")
```

Reply:
If someone else has already done this elsewhere in the code, then in my opinion it should be changed. I don't like it.